src/OXRS_WT32.h

```diff
@@ -39,6 +35,8 @@ class OXRS_WT32 : public Print
   void begin(jsonCallback config, jsonCallback command);
   void loop(void);
 
+  void setFwVersion(const char *version);
```

Comment:
Maybe add a comment to this new method, explaining how/what this version value is used for?